src/ui.rs

```diff
@@ -109,11 +110,10 @@ impl ViewState {
         self.list_state.select(Some(9));
     }
 
-    pub fn ui(&mut self, f: &mut Frame, cpu: &CPU, uart_rx: &mpsc::Receiver<char>) {
+    pub fn ui(&mut self, f: &mut Frame, cpu: &CPU, uart_rx: &mpsc::Receiver<char>, show_help: bool) {
         let size = f.size();
 
-        let block = Block::default()
-            .borders(Borders::ALL)
+        let block = Block::bordered()
             .title("Main")
             .title_alignment(Alignment::Center)
             .border_type(BorderType::Rounded);
```

Comment:
You could change the order here to move const calls up so const then non const are there. This should lead to more compiler optimizations. But its probably a very minor performance improvement.